src/api/ledger/ledger-header.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatLedgerHeader(response) {
+  const header = response.ledger;
+  return {
+    accepted: header.accepted,
+    closed: header.closed,
+    accountHash: header.account_hash,
+    closeTime: header.close_time,
+    closeTimeResolution: header.close_time_resolution,
+    closeFlags: header.close_flags,
+    ledgerHash: header.hash || header.ledger_hash,
+    ledgerVersion: parseInt(header.ledger_index || header.seqNum, 10),
+    parentLedgerHash: header.parent_hash,
+    parentCloseTime: header.parent_close_time,
+    totalDrops: header.total_coins || header.totalCoins,
+    transactionHash: header.transaction_hash
+  };
+}
+
+function getLedgerHeaderAsync(ledgerVersion, callback) {
+  validate.ledgerVersion(ledgerVersion);
```

Comment:
Hmmm, seems like it should break, but it's not. Seems like a bug in is-my-json-valid. I'll add an `if` around this so it won't break when they fix it.